source/_integrations/generic.markdown

```diff
@@ -28,23 +28,23 @@ camera:
 
 {% configuration %}
 still_image_url:
-  description: "The URL your camera serves the image on, e.g., `http://192.168.1.21:2112/`. Can be a [template](/topics/templating/)."
-  required: true
+  description: "The URL your camera serves the image on, e.g., `http://192.168.1.21:2112/`. Can be a [template](/topics/templating/). At least one of still_image_url or stream_source must be provided."
+  required: true*
```

Comment:
```suggestion
  required: true
```

Reply:
Actually you could argue that `required: false` is more accurate because it's not strictly required. But that would be inaccurate as well. How can we make it more clear that only one of these is required?

Reply:
I see that it is required and optional 😅 Maybe that should be different, but @frenck knows that better than I do.

https://github.com/home-assistant/core/pull/62611/files#diff-88b2128001e215022b56c47184199f60a68a7648d0e345490d6a2a85ac1c6b10R47-R48

Reply:
On Discord it was suggested to make them Optional as in https://www.home-assistant.io/integrations/freedns/#configuration-variables